### PR TITLE
Prepare 1.2.0

### PR DESCRIPTION
**Pull request by globin**
ping @fgrehm

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,14 +10,19 @@ IMPROVEMENTS:
...
 [GH-355]: https://github.com/fgrehm/vagrant-lxc/pull/355
 [GH-357]: https://github.com/fgrehm/vagrant-lxc/pull/357
 [GH-359]: https://github.com/fgrehm/vagrant-lxc/issues/359
 [GH-361]: https://github.com/fgrehm/vagrant-lxc/pull/361
+[GH-376]: https://github.com/fgrehm/vagrant-lxc/pull/376
+[GH-383]: https://github.com/fgrehm/vagrant-lxc/pull/383
+[GH-362]: https://github.com/fgrehm/vagrant-lxc/pull/362
 
-## [1.1.0](https://github.com/fgrehm/vagrant-lxc/compare/v1.0.1...1.1.0) (Jan 14, 2015)
+## [1.1.0](https://github.com/fgrehm/vagrant-lxc/compare/v1.0.1...v1.1.0) (Jan 14, 2015)
```
Comment (fgrehm) [inline CHANGELOG.md]: Good catch :blush:

**Comment by fgrehm** [comment]
Awesome! Should I add the email from your public GH profile as a collaborator on RubyGems.org so you can cut the release? :beers:

**Comment by fgrehm** [comment]
@globin LGTM! Which email should I add as a collaborator on RubyGems.org so you can cut the release? :beers:

**Comment by globin** [comment]
Yeah fine :+1:

**Comment by fgrehm** [comment]
@globin Have you created your account on RubyGems.org already? I'm getting a `Owner could not be found.
` when I try to use the email from your GH profile :cry:

**Comment by globin** [comment]
Have an account now

**Comment by fgrehm** [comment]
@globin There you go! IIRC you should be a `rake release` away from making it happen!

Thanks a lot to everyone involved in this release :beers:

**Pull request merged**